cli.py

```diff
@@ -21,33 +22,29 @@
 
 
 def main():
-    argument = sys.argv[1]
+    url = sys.argv[1]
```

Comment:
Before I supported reading the workflow from disk. I'm dropping that support here.